src/modules/interfaceAPI.js

```diff
@@ -0,0 +1,25 @@
+import { readFromForm, populateScoresTable, showMessage } from './appCRUD.js';
+
+const gameId = '/xAnLxnpCzCYPPY9fdvNC';
+const baseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+const scoreUrl = `${baseUrl}games${gameId}/scores/`;
+
+export const fetchFromAPI = async () => {
+  const scores = await fetch(scoreUrl).then((response) => response.json()).then((json) => json);
```

Comment:
[OPTIONAL] I think you are mixing two things performing the same function. As a requirement for this project you must use `async` and `await`. But you don't need `then` since you have used them.